apps/docs/content/guides/deployment/branching.mdx

```diff
@@ -638,6 +638,21 @@ A deployment might fail for various reasons, including invalid SQL statements an
 
 To check the error message, see the Supabase workflow run for your branch under the [View logs](/dashboard/project/_/branches) section.
 
+### Network restrictions
+
+If you enable [network restrictions](/docs/guides/platform/network-restrictions) on your project, the branching cluster will be blocked from connecting to your project by default. This often results in database connection failures when migrating your production project after merging a development branch.
+
+The workaround is to explicitly allow the IPv6 CIDR range of the branching cluster in your project's [database settings](https://supabase.com/dashboard/project/_/settings/database) page: `2600:1f18:2b7d:f600::/56`
+
+<Image
+  alt="Network restrictions to allow connections from branching cluster"
+  src={{
+    dark: '/docs/img/guides/branching/cidr-dark.png',
+    light: '/docs/img/guides/branching/cidr-light.png',
+  }}
+  zoomable
+/>
```

Comment:
@charislam qq: how do I resize this screenshot so it doesn't take up the full screen width on desktop?

Reply:
https://supabase.slack.com/archives/C023E4L60R3/p1743514864791269?thread_ts=1743458358.043619&channel=C023E4L60R3&message_ts=1743514864.791269